test/Microsoft.Dnx.Host.Tests/RuntimeEnvironmentTests.cs

```diff
@@ -35,12 +35,12 @@ public void RuntimeEnvironment_InitializedCorrectly()
 
         // Test RID generation
         [Theory]
-        [InlineData("Windows", "7.0", "x86", "win7-x86")] // 1234 => We only care about major and minor
-        [InlineData("Windows", "7.0", "x64", "win7-x64")]
-        [InlineData("Windows", "8.0", "x86", "win8-x86")]
-        [InlineData("Windows", "8.0", "x64", "win8-x64")]
-        [InlineData("Windows", "8.1", "x86", "win81-x86")]
-        [InlineData("Windows", "8.1", "x64", "win81-x64")]
+        [InlineData("Windows", "6.1", "x86", "win7-x86")] // 1234 => We only care about major and minor
```

Comment:
Long time ago this test looked like this: 
https://github.com/aspnet/dnx/commit/91bfcb2beae37e9848988d957757f02188d38a33#diff-e55dc1e04a00ac49a7639d3a48ea217dL36

I think we should remove the comment - it is no longer understandable